TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    public partial class ColorPair
```

Comment:
Class name ColorPair - Represents Information Holder but has other responsibilities like "GetColrFromPairNumber","GetColorIndex","GetPairNumberFromColor"

What is the significance of "Partial classes"

Reply:
thank you for making me understand the use of partial class in meeting . The code has been updated without using partial class and moved the code to separate class.